vnext/ReactUWP/packages.config

```diff
@@ -2,7 +2,7 @@
 <packages>
   <package id="boost" version="1.68.0.0" targetFramework="native" />
   <package id="Microsoft.UI.Xaml" version="2.3.191129002" targetFramework="native" />
+  <package id="ReactNative.Hermes.Windows" version="0.1.6" targetFramework="native" Condition="'$(OSS_RN)' != 'true' AND '$(USE_HERMES)' == 'true'" />
```

Comment:
If I remember correctly.. the predicates to check for e.g. USE_HERMES is true doesn't work .. as we use the oldish "package.config" way .. The predicates are supported with the PackageReference way (https://docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files ) but I had some other trouble which I don't remember exactly ...

Reply:
Essentially, the msbuild variables are not visible in package.config ..

Reply:
Fixing this will help in reducing the build times ..

Reply:
I postpone this to #3753. it's possible to cover WinUI and cppwinrt nuget too